<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Registrar</class>
 <widget class="QWidget" name="Registrar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>641</width>
     <height>558</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="error_message">
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="registrar">
        <item>
         <layout class="QVBoxLayout" name="usuario_layout">
          <item>
           <widget class="QLabel" name="usuario">
            <property name="text">
             <string>Usuario:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="usuario_edit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="nome_layout">
          <item>
           <widget class="QLabel" name="nome">
            <property name="text">
             <string>Nome:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="nome_edit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="email_layout">
          <item>
           <widget class="QLabel" name="email">
            <property name="text">
             <string>Email:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="email_edit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="endereco_layout">
          <item>
           <widget class="QLabel" name="endereco">
            <property name="text">
             <string>Endereço:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="endereco_edit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="senha_layout">
          <item>
           <widget class="QLabel" name="senha">
            <property name="text">
             <string>Senha:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="senha_edit">
            <property name="inputMethodHints">
             <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText</set>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="senha_confirmacao">
            <property name="text">
             <string>Confirma Senha:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="senha_confirmacao_edit">
            <property name="inputMethodHints">
             <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText</set>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="formaPagamento_layout">
          <item>
           <widget class="QLabel" name="formaPagamento">
            <property name="text">
             <string>Forma de Pagamento:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="formaPagamento_combo">
            <item>
             <property name="text">
              <string>Boleto</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Cartão de Débito</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Cartão de Crédito</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Pay Pal</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="tipo_layout">
          <item>
           <widget class="QLabel" name="tipo">
            <property name="text">
             <string>Tipo de Usuário</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="tipo_combo">
            <item>
             <property name="text">
              <string>Consumidor</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Ofertante</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="termos_servico">
          <property name="text">
           <string>Li e Aceito os Termos de Serviço</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Registrar</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
